content/en/real_user_monitoring/mobile_and_tv_monitoring/kotlin_multiplatform/setup.md

```diff
@@ -112,30 +110,27 @@ If you are integrating Kotlin Multiplatform library as a framework with an `embe
 {{< tabs >}}
 {{% tab "RUM" %}}
 
-1. Navigate to [**Digital Experience** > **Add an Application**][1].
+1. Navigate to [**Digital Experience** > **Add an Application**][2].
```

Comment:
@rtrieu does this have the correct link?

Reply:
Updated to [1], thanks!